browser-extension/mv3/src/service-worker/services/contextMenu.ts

```diff
@@ -1,34 +1,7 @@
 import { onVariableChange, setVariable } from "../variable";
-import extensionIconManager from "./extensionIconManager";
 import { isExtensionEnabled } from "../../utils";
-
-// TODO: fix circular dependency
-import { sendMessageToApp } from "./messageHandler";
-import { CLIENT_MESSAGES } from "common/constants";
-import { Variable } from "common/types";
-
-enum MenuItem {
-  TOGGLE_ACTIVATION_STATUS = "toggle-activation-status",
-}
-
-enum ToggleActivationStatusLabel {
-  ACTIVATE = "Activate Requestly",
-  DEACTIVATE = "Deactivate Requestly",
-}
-
-export const updateActivationStatus = (isExtensionEnabled: boolean) => {
-  chrome.contextMenus.update(MenuItem.TOGGLE_ACTIVATION_STATUS, {
-    title: isExtensionEnabled ? ToggleActivationStatusLabel.DEACTIVATE : ToggleActivationStatusLabel.ACTIVATE,
-  });
-
-  if (isExtensionEnabled) {
-    extensionIconManager.markExtensionEnabled();
-  } else {
-    extensionIconManager.markExtensionDisabled();
-  }
-
-  sendMessageToApp({ action: CLIENT_MESSAGES.NOTIFY_EXTENSION_STATUS_UPDATED, isExtensionEnabled });
```

Comment:
Instead of moving all this, can moving out `sendMessageToApp` to `toggleExtensionStatus` fix this?

Reply:
Nope. `utils` uses `updateActivationStatus` from `contextMenu` while `contextMenu` uses `sendMessageToApp` from `utils`

Thats why its important to move both of them

Reply:
But sendMessageToApp is in messageHandler

Reply:
`utils` <-> `messageHandler` dependency will be present because `updateActivationStatus` depends on `sendMessageToApp`

Reply:
Actually utils shouldn't be importing from `contextMenu` or `messageHandler`. Thats why its needed to move both